### ISSUE TITLE
Add BlockDeviceMapping to AWS LaunchConfiguration

### ISSUE THREAD

**Issue by pdeyhim** [mitchellh/goamz#196]
Ability to add BlockDeviceMapping is missing in launchconfiguration implementation.

**Comment by jschneiderhan** [comment]
Yes - this is a requirement for something I am working on. I'm going to try and add it/submit a PR today.

**Comment by jschneiderhan** [comment]
https://github.com/mitchellh/goamz/pull/210
